src/index.ts

```diff
@@ -5,8 +5,8 @@ import * as util from "util";
 import * as c from "ansi-colors";
 import { ReportOptions } from "istanbul-reports";
 import { reportCoverage } from "./coverage";
-import { createTestRunner, MochaReporterType } from "./runner";
 import { formatInterpreterError } from "./util";
+import { createTestRunner, MochaReporterType } from "./runner";
```

Comment:
now this needs to be `ReporterType` to support jest reporters as well, isn't it?